### PR TITLE
Hello Microverse

### PR DESCRIPTION
**Pull request by shelly-githinji**
In this project the following has been done: 
- Added a html file
- Added an index.css
- Added a test.md file
- Added npm packages

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,17 @@
...
+## Built With
+
+- HTML
+- CSS
+
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
```
Comment (aimemalaika) [inline README.md]: - [ ] consider adding more description to your readme. like social media links, project screenshots. Consider using the [Microverse readme template](https://github.com/microverseinc/readme-template).

**Review by aimemalaika — CHANGES_REQUESTED** [review]
Hi @shelly8, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

the-simpsons-homer-simpson

## Highlights
- Nice HTML structure ✅ 
- meaningful commit message ✅ 

## Required Changes ♻️

- [ ] Consider adding a descriptive summary of changes made to your pull request
- [ ] Kindly add a descriptive description on your pull request, like what all you have worked with to this branch.
- [ ] Kindly make sure that all linters passed
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by BenMukebo — APPROVED** [review]
# STATUS: Approved  🎉 

## Hi *** @shelly8 , *** 👋 

Your project is complete!  There is nothing else to say other than... it's time to merge it
Congratulations! 🎉  🎉 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by shelly-githinji** [comment]
Good day to you  Aime malaika  . I thank you for reviewing my code and
showing me where I need to make the necessary changes.

**Comment by shelly-githinji** [comment]
Good day to you Ben Mukebo. Thank you for reviewing my code.

**Pull request merged**